### PR TITLE
delete hapi negotiator and simplify response type content

### PR DESCRIPTION
**Pull request by SimonLab**
see #373

### CODE REVIEW

File: routes/html-content.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/**
+* Return true if the request ask for html content, false otherwise
+*/
+module.exports = function (request) {
+  var htmlContent = request.headers.accept && request.headers.accept.indexOf('text/html') > -1;
+  return Boolean(htmlContent);
```
Comment (Danwhy) [inline routes/html-content.js]: We should probably return HTML as the default, check for a JSON accept header and return JSON then. I know the twitter crawler bot doesn't send a header, but expects HTML as the response

File: routes/autocomplete.js [hunk 2]
```diff
@@ -2,25 +2,14 @@ const Joi = require('joi');
...
-            const queryParams = Object.assign({}, request.params, request.query);
-
-            autocomplete(elastic, queryParams, (err, results) => {
-              if (err) return reply(Boom.serverUnavailable(err));
-              reply(autocompleteResultsToJsonApi(queryParams, results, config));
-            });
-          }
-        }
-      }
+      'hapi-negotiator': false
```
Comment (Danwhy) [inline routes/autocomplete.js]: Shouldn't we delete this altogether?

Comment (SimonLab) [inline routes/autocomplete.js]: :+1:

**Comment by SimonLab** [comment]
@Danwhy @nelsonic this PR contains the last changes requested by @Danwhy and is ready for review

**Comment by nelsonic** [comment]
@SimonLab code looks good and your `route-helpers/content-type.js` file _appears_ to be _simpler_ than using `hapi negotiator` ... any idea why Codecov thinks we are missing a line of coverage...?

**Comment by SimonLab** [comment]
@nelsonic the line missing is a test on a error response, @Danwhy is working on it on the #426 PR and should be ok when this PR is merged.

**Comment by nelsonic** [comment]
@SimonLab Ok, in that case, let's get this change merged in now and get coverage back up in #426 👍

**Pull request merged**